src/view.js

```diff
@@ -61,6 +61,6 @@ export default View => class ReduxElmView extends Component {
   }
 
   render() {
-    return createElement(View, { ...this.props, dispatch: this.dispatch });
+    return createEagerElement(View, { ...this.props, dispatch: this.dispatch });
```

Comment:
The name of the function in the library has changed... no functional changes though.